articles/install-guide/update.md

```diff
@@ -16,7 +16,53 @@ This article assumes that you already have the QDK installed. If you are install
 
 The update steps depend on your development environment. Choose your environment from the following sections.
 
-## Python
+## Updating Q# Projects 
+
+1. First, install the latest version of the [.NET Core SDK 3.0](https://dotnet.microsoft.com/download) and run the following command in the command prompt:
+```bash
+dotnet --version
+```
+ Verify the output is 3.0.100 or higher, then follow the instructions below for your setup.
+
+### In Visual Studio
+ 
+ 1. Run Build > Clean Solution 
+ 1. Update the target framework in each of your .csproj files to netcoreapp3.0 (or netstandard2.1 if it is a library project)
```

Comment:
I think we might need to be more precise here?

Reply:
I don't think so...we're not teaching people how to edit project files, just giving the specific instructions needed for our product. I don't want to overload the update instructions, they already feel quite long.